ide-support/revliburl.livecodescript

```diff
@@ -513,36 +513,46 @@ on ulGetFormat pUrl,pCount
          put _proxyForURL(pURL) into laUrlProxy[pURL]
          ulLogIt "Proxy for URL:" && laUrlProxy[pURL] & cr
          
-         -- CW-2016-06-11: [[ External driver support ]] Bypass URL/Proxy manipulation for external drivers.
-         -- MM-2014-02-27: [[ HTTPS Proxy ]] We now support fetching https URLs through a proxy.
-         if laUrlProxy[pURL] is not empty and lvExtDriver is empty then
-            ## for now don't allow https connections through proxies
-            ## will look at this again later
-            put tHost into tURLHost
-            put laUrlProxy[pURL] into tHost
-            replace "http://" with "" in tHost ##not sure if this is possible but just in case
-            if tPort <> empty then
-               put tPort into tUrlPort ##save this for setting laFilename below
-            else if tPre is "https" then
-               put ":443" into tUrlPort
-            else
-               put ":80" into tUrlPort
-            end if
-            #get the proxy port
-            put the itemdel into tSavedDel
-            set the itemdel to ":"
-            if the number of items of tHost > 1 and item -1 of tHost is a number then
-               put ":" &  item -1 of tHost into tPort
-               delete item -1 of tHost #remove port for now
-            else
-               put ":80" into tPort
+         -- CW-2016-08-14: [[ External driver support ]] Bypass URL/Proxy manipulation and async check for external drivers.
+         if lvExtDriver is empty then
+            -- CW-2016-08-14: If we are not using an external driver, asynchronous HTTP uploads are not supported.
+            if laAction[pUrl] is "putData" and laLoadReq[pUrl] is true then
```

Comment:
@techstrategies I'm a bit confused on this. Was this required because it's now possible to start an asynchronous HTTP PUT with `libURLFtpUpload`? I wonder if libURL should just fallback to synchronous rather than fail here?